plugins/unmanaged_files/unmanaged_files_model.rb

```diff
@@ -73,12 +73,12 @@ def compare_with(other)
     comparison.push(common)
   end
 
-  def has_metadata?
-    @metadata ||= has_metadata || elements.any?(&:user)
+  def contains_metadata?
+    @metadata ||= has_metadata || !!elements.any?(&:user)
   end
 
-  def file_objects?
-    @file_objects ||= elements.any?(&:file_objects)
+  def has_subdir_count?
```

Comment:
I think `has_subdir_counts?` might be more appropriate because it's a method of the list, and the list has multiple counts, one per element.